src/css/utils/_border.scss

```diff
@@ -63,3 +63,9 @@
 .inner-ring-info {
   box-shadow: inset 0 0 0 2px var(--sn-stylekit-info-color);
 }
+
+.border-bottom-1-gray {
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: var(--sn-stylekit-border-color);
+}
```

Comment:
For utility classes, we usually want to keep each class doing only one thing. So to achieve this same result, you'd combine the classes "border-b-1" and "border-gray-300"

Reply:
Done. Commit amended.